oteapi_dlite/utils/utils.py

```diff
@@ -71,3 +88,39 @@ def get_driver(mediaType=None, accessService=None, options=None) -> str:
         return ACCESSSERVICES[accessService]
 
     raise ValueError("either `mediaType` or `accessService` must be provided")
+
+
+def get_instance(
+    meta: "Union[str, dlite.Metadata]",
+    collection: "dlite.Collection",
+    routedict: "Optional[dict]" = None,
+    instance_id: "Optional[str]" = None,
+    allow_incomplete: bool = False,
+    **kwargs,
+) -> dlite.Instance:
+    """Instantiates and returns an instance of `meta`.
+
+    Arguments:
+        meta: Metadata to instantiate.  Typically its URI.
+        collection: The collection with instances and mappings.
+
+    Some less used optional arguments:
+        routedict: Dict mapping property names to route number to select for
+            the given property.  The default is to select the route with
+            lowest cost.
+        instance_id: URI of instance to create.
+        allow_incomplete: Whether to allow not populating all properties
+            of the returned instance.
+        kwargs: Additional arguments passed to dlite.mappings.instantiate().
+    """
+    ts = Triplestore(backend="collection", collection=collection)
+    inst = instantiate(
+        meta=meta,
+        instances=list(collection.get_instances()),
+        triplestore=ts,
+        routedict=routedict,
+        id=instance_id,
+        allow_incomplete=allow_incomplete,
+        **kwargs,
+    )
+    return inst
```

Comment:
```suggestion
    return instantiate(
        meta=meta,
        instances=list(collection.get_instances()),
        triplestore=ts,
        routedict=routedict,
        id=instance_id,
        allow_incomplete=allow_incomplete,
        **kwargs,
    )
```

Reply:
I prefer the original version since the introduction of the `inst` variable makes it very clear that we are returning an instance.
It also makes it easier to insert print statements before return for debugging purposes.